packaging/desktop/package_darwin.sh

```diff
@@ -13,11 +13,14 @@ save_dir=${SAVE_DIR:-}
 tmp_dir="/tmp/package_darwin/tmp"
 bucket_name=${BUCKET_NAME:-}
 run_mode="prod"
-platform="darwin"
 s3host=${S3HOST:-}
 istest=${TEST:-}
 skip_notarize=${SKIP_NOTARIZE:-}
 arch=${ARCH:-"amd64"}
+platform="darwin"
```

Comment:
This isn't right, the build machine just needs to specify `ARCH`, probably through the bot at top level.